cypress/e2e/views/applicationinventory.view.ts

```diff
@@ -13,7 +13,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-export const actionButton = "#toolbar-kebab";
+export const actionButton = "button[aria-label='kebab dropdown toggle']";
```

Comment:
You are not using this in the PR . Does it need to be updated ?

Reply:
The code changes from that file have been reverted in the latest commit.